docs/conf.py

```diff
@@ -72,8 +73,8 @@
 
 # General information about the project.
 project = "PlasmaPy"
-copyright = "2015-2020, PlasmaPy Community"
 author = "PlasmaPy Community"
+copyright = f"2015-{datetime.utcnow().year}, {author}"
```

Comment:
Good hack! :laughing:

Reply:
I actually saw it in astropy's conf.py while search for something else.  And I was, "Wow! Why have we never thought of that.  We must have it!"